README.md

```diff
@@ -1,2 +1,19 @@
-# theme-experiments
-Experimenting with themes made out of blocks.
+# WordPress Theme Experiments
+Experimenting with themes made out of blocks. 
+
+## About
+This is a monorepo housing many experiments. Typically each sub-directory is a subcontained theme that explores some aspect of creating themes using blocks / block templates.
+
+## What's Here
+To be populated by you!
```

Comment:
Let's just remove this section until we have something to put here. Presumably once we merge some more PRs, we can include links to individual README files or PRs.